### PR TITLE
Fixing some typos

### PR DESCRIPTION
**Pull request by pdube**
Fixing some typos

### CODE REVIEW

File: os/booting-on-cloudca.md [hunk 1]
```diff
@@ -141,9 +141,8 @@ coreos:
         Type=btrfs
 ```
 
-The $public_ipv4 substitution variables is not supported in cloud-config on
-cloud.ca. To use multi-region cluster, cluster configuration must be performed
-manually and port-forwarding configured.
+The $public_ipv4 substitution variable is not supported in cloud-config on
+cloud.ca. To use a multi-region cluster, the cluster and the port-forwarding must be configured manually.
```
Comment (pdion891) [inline os/booting-on-cloudca.md]: wrong
it's the cluster configuration not the cluster, Port forwarding and cluster conf are two different configuration in two different places.

File: os/booting-on-cloudca.md [hunk 2]
```diff
@@ -173,35 +172,35 @@ Before deploying CoreOS on cloud.ca, you need following:
...
+4. Enable port forwarding rule for SSH (Optional) and assign your SSH key to the new instance:
 <div class="row">
   <div class="col-lg-8 col-md-10 col-sm-8 col-xs-12">
     <img src="img/cloudca-addinstance_step3.png" class="screenshot" />
-    <div class="caption">New Instance data volume</div>
+    <div class="caption">New instance connectivity</div>
   </div>
 </div>
-5. Add your cloud-config content (Optional):
+5. Add your cloud-config content:
```
Comment (pdion891) [inline os/booting-on-cloudca.md]: it's optional.

File: os/booting-on-cloudca.md [hunk 3]
```diff
@@ -43,8 +43,8 @@ coreos:
     </div>
     <div class="tab-pane" id="beta">
       <div class="channel-info">
-        <p>The beta channel consists of promoted alpha releases.
-        To use beta channel on cloud.ca, use "CoreOS Stable" template and update cloud-config file to change the channel to beta:
+        <p>The Beta channel consists of promoted alpha releases.
+        To use the Beta channel on cloud.ca, use the "CoreOS Stable" template and update the cloud-config file to change the channel to beta:
```
Comment (pdion891) [inline os/booting-on-cloudca.md]: it's not a file we need to remove this.

File: os/booting-on-cloudca.md (hunk 1 above)
Comment (pdube) [inline os/booting-on-cloudca.md]: Yes, this doesn't imply that it is the same file

File: os/booting-on-cloudca.md (hunk 3 above)
Comment (pdube) [inline os/booting-on-cloudca.md]: Ok

File: os/booting-on-cloudca.md (hunk 2 above)
Comment (pdube) [inline os/booting-on-cloudca.md]: Is this optional, if you don't want an additional volume?

**Pull request merged**